flyway.go

```diff
@@ -68,6 +75,10 @@ func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomize
 		}
 	}
 
+	if err := parseRequest(genericContainerReq); err != nil {
+		return nil, err
+	}
```

Comment:
this is just a language thing I guess, but a parser to me is something which takes an input and parses it (converts) it to something, whereas here what we're really doing is validating the request. If it was a boolean response I tend to prefix the func with is/was, but here as it is an error result, I would generally do

```
	if err := validateRequest(genericContainerReq); err != nil {
		return nil, err
	}
```

NOTE: I don't mind really and if you prefer parse we can leave it as is

Reply:
I've used validated in the past, but starting to see parse more and more as a way to check for errors. Either name is fine for me, so feel free to change it to the one you prefer.